### DISCUSSION TITLE
下學期值得修的網頁課

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#503] [General]
![lunx擷取](https://user-images.githubusercontent.com/113414249/208580663-7116fbf6-201c-4abb-aa87-063c0e5ff9ec.PNG)
![1111擷取](https://user-images.githubusercontent.com/113414249/208580669-fcf704c2-11f2-45ea-814b-f40ec5adb32f.PNG)

我們這學期有學到一些html css style的框架，如何POST GET
如果想做更進一步的研讀，推爆這一堂課，保證會學到多延伸
而且這堂課 每周都有作業 是該堂課的應用 而這部分是我們一定要學會 才做得出來的
等於說若完整學完一學期 保證扎扎實實

可以幫我補充更多 以及還有什麼好課 也可以推薦一下 感謝

**Comment by NcuJerryKao** [comment]
忘記說了 這堂課採取密碼卡選課 而非分發
故是在第一堂課拿密碼卡 記得要來喔

**Comment by 32844583** [comment]
![螢幕擷取畫面 2022-12-20 222101](https://user-images.githubusercontent.com/66071134/208688595-6ca8c844-bc10-4f81-a8e9-03f26f5bdfed.jpg)
順便幫各位複習一下 POST 跟 GET 的差別，嚴格來說一般的表單可以用 GET 直接傳遞，而需要保密的資料必須用 POST 來處理，像是會員登入的帳號密碼。

**Comment by manipulateed** [comment]
看這個課綱感覺較內容會非常扎實欸🫣,而且也透過考試去讓學生主動去學習，很值得可考慮～

**Comment by brian431** [comment]
看起來是門值得一修的課，謝謝分享。

**Comment by gnsJhenJie** [comment]
聽說這堂課教的東西很多，如果有想學的可以修修看。

**Comment by williamyen0623** [comment]
看課程內容感覺很扎實！而且AI第一次作業剛好有學到一點html css跟js，馬上銜接應該很不錯！謝謝分享！

**Comment by Ben6uo** [comment]
感覺是一門很硬的課 我需要更努力了

**Comment by liangyinglly** [comment]
感覺外系不太好拿到密碼卡

**Comment by skyling0299** [comment]
這邊推資管系
網頁程式設計，雖然不好拿密碼卡，但老師人很好
教的東西比較偏向讓大家寫js或php等等，也可以從中學css的應用唷

**Comment by HsuChen88** [comment]
扎實，若對自己的網頁有理想的人，先預祝這些同學，睡眠充足。

**Reply by chia-kai-chang** [reply]
有八卦嗎? 想聽~

**Comment by boson13579** [comment]
推推資工系陳映儂教授的程式設計!!! 儂儂給分很甜，上課內容是c++為主，程式新手也可以輕鬆學會，只要每個禮拜作業有教(很簡單的扣)，人人都是95分。

**Comment by xCk27x** [comment]
中央的課程裡好像沒有網頁設計的進階課程
這樣在學完這部分知識之後還是要花大量的時間自學才能到能夠實用的地步

**Comment by bww0401** [comment]
感謝分享，雖然數學系應該拿不到密碼卡...

**Comment by Aerofe** [comment]
這門課之前有修過，教的東西非常多，課程也蠻緊湊的，如果想要教的都學會的話一定要額外花課外時間去研究，練習

**Comment by ying0607** [comment]
感覺是很紥實的課，謝謝分享！

**Comment by 0212Jerry** [comment]
感謝分享，有人能整理真好，之前就有在觀望一些課程，但又不敢貿然的選下去

**Comment by Vincent20030314** [comment]
這堂課我有修過!雖然有點硬，但是分數不會太低也能學到很多東西。
